src/app/+search-page/search-filters/search-filter/search-fixed-filter.service.spec.ts

```diff
@@ -1,17 +1,15 @@
-import { SearchFixedFilterService } from './search-fixed-filter.service';
-import { RouteService } from '../../../shared/services/route.service';
-import { RequestService } from '../../../core/data/request.service';
-import { HALEndpointService } from '../../../core/shared/hal-endpoint.service';
-import { of as observableOf } from 'rxjs';
-import { RequestEntry } from '../../../core/data/request.reducer';
-import { FilteredDiscoveryQueryResponse, RestResponse } from '../../../core/cache/response.models';
+import {SearchFixedFilterService} from './search-fixed-filter.service';
```

Comment:
Could you revert spaces between curly brackets?

Reply:
Reset original spacing